.gitignore

```diff
@@ -17,13 +17,16 @@ expected/
 notes/
 baserom/
 docs/doxygen/
+cache/
+vc/
```

Comment:
that folder doesn't looks created or used at all, you should probably delete this line

Reply:
ill address this in a second